src/panels/DetectorPanel.ts

```diff
@@ -6,7 +6,7 @@ const meta = require('../../package.json');
 
 export class DetectorPanel extends BasePanel<"detector"> {
     constructor(extensionUri: Uri) {
-        super(extensionUri, "detector");
+        super(extensionUri, "detector", {});
```

Comment:
this panel doesn't have webview post messages ?

Reply:
That's right - and if it did, the above would cause a TypeScript compilation error